.github/workflows/linux_riscv_xuantie.yml

```diff
@@ -0,0 +1,256 @@
+name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
```

Comment:
I'd suggest to add version of rvv to the job name because we have 2 cmake toolchains files for different vector instruction sets : for rvv0.7.1 and rvv1.0. In the current dockerfile we cross-compile OpenVINO for RVV0.7.1.

```suggestion
name: Linux RISC-V RVV 0.7.1 with Xuantie (Ubuntu 22.04, Python 3.10)
```